contentcuration/contentcuration/frontend/channelEdit/components/edit/EditModal.vue

```diff
@@ -399,7 +399,11 @@
 
                 if (completeCheck !== node.complete) {
                   validationPromises.push(
-                    vm.updateContentNode({ id: nodeId, complete: completeCheck })
```

Comment:
I guess technically, we don't need to rerun the complete check here, because we already ran it above?